frontend/apps/web/app/(mgmt)/[account]/runs/[id]/page.tsx

```diff
@@ -435,18 +466,18 @@ function StatCard(props: StatCardProps): ReactElement {
   );
 }
 
-function getDuration(dateTimeValue2?: Date, dateTimeValue1?: Date): string {
-  if (!dateTimeValue1 || !dateTimeValue2) {
+function getDuration(completedAt?: Date, startedAt?: Date): string {
+  if (!startedAt || !completedAt) {
     return '';
   }
-  var differenceValue =
-    (dateTimeValue2.getTime() - dateTimeValue1.getTime()) / 1000;
-  const minutes = Math.abs(Math.round(differenceValue / 60));
-  const seconds = Math.round(differenceValue % 60);
+  var diff = (completedAt.getTime() - startedAt.getTime()) / 1000;
+  const minutes = Math.floor(diff / 60);
+  const seconds = Math.round(diff % 60);
+  // handle plural minutes and seconds
   if (minutes === 0) {
-    return `${seconds} seconds`;
+    return `${seconds} second${seconds !== 1 ? 's' : ''}`;
   }
-  return `${minutes} minutes ${seconds} seconds`;
+  return `${minutes} minute${minutes > 1 ? 's' : ''} ${seconds} second${seconds !== 1 ? 's' : ''}`;
```

Comment:
Could we replace all of this logic with [formatDuration](https://date-fns.org/v3.6.0/docs/formatDuration) ? I think that does exactly what we want here, but better.